### PR TITLE
[Task #51820] [Admin] Thêm mới sản phẩm

### PR DESCRIPTION
**Pull request by thonght-3051**
## Related Tickets
- [#51820](https://edu-redmine.sun-asterisk.vn/issues/51820)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

### CODE REVIEW

**Comment by thonght-3051** [comment]
ready

File: app/assets/stylesheets/admin/product.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.add_fields {
+    float: right;
+}
+
+.product_image {
+    width: 45px;
```
Comment (ghost) [inline app/assets/stylesheets/admin/product.scss]: indent 2 space

**Comment by thonght-3051** [comment]
ready

**Comment by thonght-3051** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**